hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/types/VolumeMetadata.java

```diff
@@ -0,0 +1,184 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hadoop.ozone.recon.api.types;
+
+import com.google.common.base.Preconditions;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import java.util.List;
+
+/**
+ * Metadata object represents one volume in the object store.
+ */
+@XmlAccessorType(XmlAccessType.FIELD)
+public final class VolumeMetadata {
```

Comment:
@ivandika3  Was it not possible to use existing OmVolumeArgs or VolumeObjectDBInfo (already in Recon) object ?

Reply:
Yes, I think it is possible. In our own cluster, we have not incorporated the `VolumeObjectDBInfo` and `BucketObjectDBInfo`, therefore we need to write a new DTO for this. I will try to incorporate the existing volume and bucket classes.

Reply:
I have removed `VolumeMetadata` uses `VolumeObjectDBInfo` and modifying `ObjectDBInfo#acls` to use `AclMetadata` instead of `OzoneAcl`.